dali/test/python/test_tensor_cupy.py

```diff
@@ -0,0 +1,59 @@
+# Copyright (c) 2021, NVIDIA CORPORATION. All rights reserved.
```

Comment:
Why not move this tests to test_backend_impl_gpu.py ?

Reply:
Done